inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -683,9 +683,6 @@
                     "id": {
                         "type": "string"
                     },
-                    "set": {
```

Comment:
I think this field is in the `dojson` definition so being imported?

If the problem is that the value is an array or viceversa maybe we can do  `"type": ["array", "string"]`

Ideally it can be fixed on import time https://github.com/inspirehep/inspire-next/blob/invenio3/inspirehep/dojson/common/base.py#L109 to make sure the field has a string inside (or an array) - @kaplun do you know the type of this field?